runtime-services-modeling/decisions-with-dmn.adoc

```diff
@@ -519,13 +515,13 @@ Boxed expressions in DMN are tables that you use to define the underlying logic
 
 The following are the types of DMN boxed expressions:
 
-* Decision tables
-* Literal expressions
-* Contexts
-* Relations
-* Functions
-* Invocations
-* Lists
+* link:#con-dmn-decision-tables[Decision tables]
+* link:#con-dmn-literal-expressions[Literal expressions]
+* link:#con-dmn-context-expressions[Contexts]
+* link:#con-dmn-relation-expressions[Relations]
+* link:#con-dmn-function-expressions[Functions]
+* link:#con-dmn-invocation-expressions[Invocations]
+* link:#con-dmn-list-expressions[Lists]
```

Comment:
These should be xrefs. That said, I'm not 100% sure if internal xrefs are going to work correctly in IBM Docs. They might be doing some link rewriting, and I don't know if it'll work well.